tools/tidb-binlog-kafka.md

```diff
@@ -114,8 +114,10 @@ cd tidb-binlog-latest-linux-amd64
 ```
 # binlog trigger
 enable_binlog = True
-# ZooKeeper address of Kafka cluster, example:
+# zookeeper address of Kafka cluster, example:
```

Comment:
The "zookeeper" here is a description, not within code or command. I think we should keep it as "ZooKeeper" (the official format)?

Please also modify other similar ones in this PR.